src/main/java/org/terasology/web/webSocket/WsHandler.java

```diff
@@ -14,37 +14,67 @@
  * limitations under the License.
  */
 
-package org.terasology.web.servlet;
+package org.terasology.web.webSocket;
 
 import java.io.IOException;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
 import org.eclipse.jetty.websocket.api.Session;
 import org.eclipse.jetty.websocket.api.WebSocketAdapter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.terasology.web.io.ActionResult;
+import org.terasology.web.io.JsonSession;
 
 /**
  * Manages one websocket session
  */
-public class EventSocket extends WebSocketAdapter {
+public class WsHandler extends WebSocketAdapter {
 
-    private static final Logger logger = LoggerFactory.getLogger(EventSocket.class);
+    private static final Logger logger = LoggerFactory.getLogger(WsHandler.class);
+    private static final Gson GSON = new GsonBuilder().disableHtmlEscaping().create();
```

Comment:
You mean shared between `WebSocketHandler` and `GsonMessageBodyHandler`? Not sure about that since they serve different purposes.
- The instance in `WebSocketHandler` is used for "container" JSON objects, like `ClientMessage`, which encapsulate objects to send to `JsonSession` for further processing and add some fields specific to the WebSocket (like messageType, which in HTTP is not necessary because the information is in the URL).
- The instance in `GsonMessageBodyHandler` is used for incoming and outgoing HTTP messages; has the custom type adapters for Version and Name that in my understanding shouldn't be necessary for WebSocket (but maybe aren't necessary here anymore, too)?